### DISCUSSION TITLE
Version Specific Documentation

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#19572] [Feature Requests]
### Summary

Is it possible to get version specific documentation (for example v9)?

### Motivation

For all people who cant easily upgrade their directus

### Detailed Design

Selection of the specific version (major version would be enough)

### Must Have:
- Major version

### Should Have:
- Minor version

### Could Have:
- Patch

### Drawbacks

Just some informations of the documentation?

### Alternatives

Using way back machine?

### Adoption Strategy

Maybe using git for version control?

**Comment by rijkvanzanten** [comment]
The docs are in the repo, so you can find the old documentation by opening the older tags, like https://github.com/directus/directus/tree/v10.0.0. We won't maintain docs for outdated versions on the website version at this time
